### PR TITLE
Send email via api

### PR DESCRIPTION
**Pull request by nocommentcode**
This PR implements the sending of the game pgn email via the tornelo endpoint

### CODE REVIEW

File: packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  const [inScanQRMode, setInScanQRMode] = useState(true);
+
+  const [arbiterPin, setArbiterPin] = useState(arbiterInfo?.pin ?? '');
+  const [pgnUrl, setpgnUrl] = useState(arbiterInfo?.broadcastUrl ?? '');
+  const [arbiterEmailSecret, setArbiterEmailSecret] = useState(
+    arbiterInfo?.arbiterToken ?? '',
+  );
+  const [divisionId, setdivisionId] = useState(arbiterInfo?.divisionId ?? '');
+  const [arbiterId, setArbiterId] = useState(arbiterInfo?.userId ?? '');
+  const [eventName, seteventName] = useState(arbiterInfo?.eventName ?? '');
```
Comment (gwenllian1) [inline packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx:42]: is there a reason setpgnUrl, setdivisionId, and seteventName don't have the second word capitalized?

File: packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+    if (validateArbiterInfo(newArbiterInfo)) {
+      await setArbiterInfo(newArbiterInfo);
+    }
+  };
+  return (
+    <>
+      <Sheet title="Edit Event" dismiss={dismiss} visible={display}>
+        <View
+          style={{ display: 'flex', flexDirection: 'row', paddingBottom: 20 }}>
```
Comment (gwenllian1) [inline packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx:111]: could some of the style stuff go in the style.ts file?

File: packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+          </PrimaryText>
+        </View>
+        {/* QR Code Entry */}
+        {inScanQRMode && (
+          <View style={styles.editEventSheetSection}>
+            {/* Placeholder - QR Code scanning camera section will go here call confirmArbiterInfo() once the qr code has scanned and everything will be saved!*/}
+            <View
+              style={{
+                backgroundColor: colours.grey10,
+                padding: 30,
```
Comment (gwenllian1) [inline packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx:139]: Again here, should the style information be in style.ts?

File: packages/TorneloScoresheet/src/util/env.ts [hunk 4]
```diff
@@ -1,4 +1,8 @@
-import { TORNELO_URL, MAX_PAST_GAME_STORED } from '@env';
+import { TORNELO_URL, MAX_PAST_GAME_STORED, EMAIL_API_ENDPOINT } from '@env';
 
 export const torneloUrl = TORNELO_URL;
 export const maxPastGameStored = MAX_PAST_GAME_STORED;
+export const emailEndpoint = (arbiterId: string): string => {
+  console.log(EMAIL_API_ENDPOINT);
```
Comment (gwenllian1) [inline packages/TorneloScoresheet/src/util/env.ts:6]: there are a few console.log statements here and in emailUtils. Are they meant to be there?

**Review by gwenllian1 — CHANGES_REQUESTED** [review]
Looks good :) I left a couple of comments about a few small things if you think they need to be changed.

File: packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+            size={26}
+            weight={FontWeight.Medium}
+            style={styles.instructions}>
+            Go to{' '}
+            <Link
+              style={styles.instructionsLink}
+              label="tornelo.com"
+              link={torneloUrl}
+            />{' '}
+            and either generate a QR code or manually edit the fields here
```
Comment (gwenllian1) [inline packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx:129]: It isn't clear how to actually manually edit the fields, I had to watch the video on the other pr to see how to do it

File: packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx (hunk 3 above)
Comment (nocommentcode) [reply packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx:139]: This is a placeholder and will be replaced once bryn implements the qr code stuff, so i didnt bother

File: packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx (hunk 5 above)
Comment (nocommentcode) [reply packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx:129]: yeh i was worried about that, I changed it to look like this: 

image

image

Is that better?

**Comment by nocommentcode** [comment]
Ok i fixed the changes for the arbiter info stuff, btw @gwenllian1 , i think you were reviewing the wrong PR, this pr is only about the email sending stuff, but its rebased onto the other brach because i need the arbiter info stuff, let's get #161 approved first, then ill rebase this one onto main and then we can merge this one!

**Review by gwenllian1 — APPROVED** [review]

**Pull request merged**